<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondW</class>
 <widget class="QDialog" name="SecondW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>751</width>
     <height>541</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>501</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/1.jpeg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="question">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>У анимационной студии &quot;GHIBLI&quot; есть логотип. Кто на нем изображен?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="image2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>471</width>
       <height>471</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/castle.jpeg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="question2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Что стало прообразом ходячего замка Хаула?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="image3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/totoro.jpeg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="question3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Хранителем чего было огромное мохнатое существо из &quot;Мой сосед - Тоторо&quot;?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="image4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/mononoke.jpeg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="question4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Вспомните, сколько братьев-волков было у дикарки Сан?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="question5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>В аниме «Навсикая из долины ветров» не было духов и призраков, но зато были гигантские насекомые. Как называли вот этих жуков?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/bugs.jpeg</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="question6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>Помните Кики из «Ведьминой службы доставки»? У нее был классный кот. Как его звали?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/witch.jpeg</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QLabel" name="question7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Внимание: сложный вопрос! Реклама чего была на автобусе, в который врезалась Кики, только прибыв в чужой город на стажировку?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/bus.jpeg</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QLabel" name="question8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Кость какой рыбы натолкнула авиаконструктора Дзиро на идею, как сконструировать идеальное крыло для самолета?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/fish.jpeg</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_9">
    <widget class="QLabel" name="question9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>А на какой по счету станции жила ведьма Дзиниба, сестра Юбабы?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image9">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/twin.jpeg</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_10">
    <widget class="QLabel" name="question10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>470</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Перейдем к «Небесному замку Лапута». Из чего изготовлен кулон Ситы?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image10">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>501</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myPro.qrc">:/stone.jpeg</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>710</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>35</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 243, 243);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="passButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>700</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пропустить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>700</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>501</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>640</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>690</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(108, 206, 110, 213);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Результат</string>
   </property>
  </widget>
  <widget class="QLabel" name="points">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>score</zorder>
  <zorder>passButton</zorder>
  <zorder>answerButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>result</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>points</zorder>
 </widget>
 <resources>
  <include location="myPro.qrc"/>
 </resources>
 <connections/>
</ui>
